app/models/custom_form.py

```diff
@@ -1,6 +1,19 @@
 import json
+import binascii
+import os
 
 from app.models import db
+from app.api.helpers.db import get_count
+
+
+def get_new_custom_form_identifier(length=8):
```

Comment:
And also, it should be unique among rows of the same form within an event. (ie. you should set a multicolumn unique comprising of event_id, field_identifier and form)